bin/load-mocks

```diff
@@ -1088,6 +1127,15 @@ def create_mock_transactions(
                 spans=[parent_span] + spans,
             )
 
+        def load_performance_issues():
+            print(f"    > Loading performance issues data")  # NOQA
+            print(f"    > Loading n plus one issue")  # NOQA
+            load_n_plus_one_issue()
+            print(f"    > Loading consecutive db issue")  # NOQA
+            load_consecutive_db_issue()
+            print(f"    > Loading uncompressed asset issue")  # NOQA
+            load_uncompressed_asset_issue()
```

Comment:
A bit of a nit,
Any reason this was moved below all the performance issue loading functions?

Personally I liked this at the top, I found it was a tad easier to navigate to specific performance issue loading functions using the IDE (less scrolling), and I thought it was more convenient being right after the `if load_performance_issues:`, the functions seemed to be ordered based on when they are executed.

Reply:
Oh I don't really scroll or really mind either way. Very minor preference for me, I just usually have it so functions are closer to where they are being called. This function is called in a line just below. 🤷

Reply:
Yeah, idc too much, it was mostly when I was first looking at it.